src/platform/site-wide/sass/shame.scss

```diff
@@ -571,3 +570,13 @@ img[data-srcset] {
   border-top-left-radius: 0;
   border-bottom-left-radius: 0;
 }
+
+.sco-card {
+  background-color: #e1f3f8;
```

Comment:
Can this use the design system utilities instead of adding this to the global CSS? 

- for the [background-color](https://design.va.gov/utilities/background-color), `vads-u-background-color--primary-alt-lightest` can be used right in the markup or `$color-primary-alt-lightest` can be used in the css in place of the hex value 
- padding [utility](https://design.va.gov/utilities/padding#padding-on-all-sides) 
- margin-top [utilty](https://design.va.gov/utilities/margins#margin-top)
- text-decoration [utility](https://design.va.gov/utilities/text-decoration)

Reply:
Hi @rianfowler, thanks for the suggestions. I will go ahead and make those changes in the markdown and should be able to close this PR afterwards.